### ISSUE TITLE
Nan occur during training

### ISSUE THREAD

**Issue by ZhenhuiTang** [WeijingShi/Point-GNN#12]
Hi Weijing, I have some questions during training and evaluation.
As shown below,  the reg loss get nan from the first step. I just change the training dir path and set GPU and batch are 1 respectively.
![image](https://user-images.githubusercontent.com/26131661/79188803-a4788000-7e52-11ea-9a46-8f4e1db2b843.png)

**Comment by ZhenhuiTang** [comment]
Besides, i have some puzzle about the result of evaluation:
![image](https://user-images.githubusercontent.com/26131661/79189181-abec5900-7e53-11ea-9ae1-3703994f4ee3.png)
![image](https://user-images.githubusercontent.com/26131661/79189190-b149a380-7e53-11ea-9fcd-12d114def6c7.png)
![image](https://user-images.githubusercontent.com/26131661/79189196-b4dd2a80-7e53-11ea-8185-244359de5b00.png)

1. Dose the "car_dectection_AP" mean the AP (IOU=0.5) in Image plane?
2. what the "car_orientation_AOS" mean ?
3. Dose the "car_detection_BEV_AP"is with IOU=0.7?  
4. Dose the "car_detection_3D_AP"is with IOU=0.7?  
5. where is AOS ?
6. where is the evaluation result to cyclists and pedestrians?

best regards.

**Comment by WeijingShi** [comment]
Hi, Nan is common when the learning rate is too large. You are using batch_size=1 so the gradient is noisier and you might need a smaller learning rate. You can try to reduce it and see if you still get Nan.  In my experiment, I also found gradient clipping and the cyclical learning rate are helpful. 

Does the "car_detection_BEV_AP"is with IOU=0.7? yes.
Does the "car_detection_3D_AP"is with IOU=0.7? yes.
We do not use AOS (part of the AVOD paper) or the 2D AP (IOU=0.7).  Please refer to [asharakeh](https://github.com/asharakeh/kitti_native_evaluation/tree/f2f70059e643556bb825166fdc4dfaf7d1a798a0) 's implementation for what they mean. 

The cyclists and pedestrians are detected by a separated [model](../blob/master/checkpoints/ped_cyl_auto_T3_trainval)

Thanks,

**Comment by ZhenhuiTang** [comment]
> Hi, Nan is common when the learning rate is too large. You are using batch_size=1 so the gradient is noisier and you might need a smaller learning rate. You can try to reduce it and see if you still get Nan. In my experiment, I also found gradient clipping and the cyclical learning rate are helpful.
> 
> Does the "car_detection_BEV_AP"is with IOU=0.7? yes.
> Does the "car_detection_3D_AP"is with IOU=0.7? yes.
> We do not use AOS (part of the AVOD paper) or the 2D AP (IOU=0.7). Please refer to [asharakeh](https://github.com/asharakeh/kitti_native_evaluation/tree/f2f70059e643556bb825166fdc4dfaf7d1a798a0) 's implementation for what they mean.
> 
> The cyclists and pedestrians are detected by a separated [model](../blob/master/checkpoints/ped_cyl_auto_T3_trainval)
> 
> Thanks,

I'm really grateful.
